.azure-pipelines/common/publish-error-codes.yml

```diff
@@ -0,0 +1,11 @@
+#
+# Azure Pipelines step to publish error code artifacts for react on Linux, Windows, and macOS.
+#
+
+steps:
+  - task: PublishBuildArtifacts@1
+    inputs:
+      pathtoPublish: './scripts/error-codes/codes.json'
+      artifactName: ${{ parameters.artifactName }}
+      publishLocation: 'container'
```

Comment:
I think `container` is the default (and isn't required)

Reply:
removed.